react/Autosuggest/Autosuggest.demo.js

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 import Autosuggest from './Autosuggest';
 import styles from '../TextField/TextField.less';
 import classnames from 'classnames';
-import fieldMessageOptions from '../private/FieldMessage/FieldMessage.demo';
-import fieldLabelOptions from '../private/FieldLabel/FieldLabel.demo';
+import fieldMessageOptions from '../FieldMessage/FieldMessage.privateDemo';
```

Comment:
Is there a reason this can't just be `FieldMessage.demo`, like the other components?

Reply:
Because get's auto imported with `https://github.com/seek-oss/seek-style-guide/blob/2962cce3fb8fc6e9ba853c2cfedc94c3ef6a1a34/docs/src/components/App/App.js#L13` and as it's not a normal demo (just imported into other demos) it fails.

Reply:
Open to other suggestions/names?

Reply:
Ah, yeah, this is a tricky one to name. Not sure that this is "private" compared to the others. How about `demoFragment`?

Reply:
Sounds good to me, I'll switch it over 👍

Reply:
Should this PR also add a proper demo for these components, rather than just fragments?